app/views/friendships/_friends_list.html.erb

```diff
@@ -0,0 +1,39 @@
+<% obj.each do |friendship| %>
```

Comment:
`obj` is about as nondescriptive of a parameter name as you can get. That could literally be anything